interpreter/cling/include/cling/Interpreter/Interpreter.h

```diff
@@ -12,8 +12,6 @@
 
 #include "cling/Interpreter/InvocationOptions.h"
 
-#include "llvm/ADT/StringRef.h"
```

Comment:
Great! But shouldn't this also mean that we can reduce the number of headers that gets copied by the build  system?